riva/clients/nmt/riva_nmt_t2t_client.cc

```diff
@@ -90,13 +90,30 @@ translateBatch(
   }
 }
 
+int countWords(const std::string& text) {
+    int wordCount = 0;
+    bool inside_word = false;
+
+    for (char c : text) {
+        if (std::isspace(c)) {
+            inside_word = false;
+        } else if (!std::ispunct(c)) {
```

Comment:
do we need to handle punctuation symbol differently here or can just count based on space? if we have punctuation symbols preceded by space, that's going as a new word for the model to translate, so should be accounted in total words.